gridlayout.h

```diff
@@ -42,6 +42,8 @@ class Grid
 
 public:
     // Constructor:
+    // Default Constructor
+    Grid(){}
```

Comment:
This constructor may cause issues as most of the properties are private and a handful cannot be modified outside of the class

Reply:
Default constructor is required to allow the object to be a class property.